emp-churn-step-by-step/README.md

```diff
@@ -0,0 +1,41 @@
+# Getting Started with H2O Wave
+
+This project was bootstrapped with `wave init` command.
+
+## Running the app
+
+Make sure you have activated a Python virtual environment with `h2o-wave` installed.
+
+If you haven't created a python env yet, simply run the following command (assuming Python 3.7 is installed properly).
+
+For MacOS / Linux:
+
+```sh
+python3 -m venv venv
+source venv/bin/activate
+pip install h2o-wave
+```
+
+For Windows:
+
+```sh
+python3 -m venv venv
+venv\Scripts\activate.bat
+pip install h2o-wave
+```
+
+Once the virtual environment is setup and active, run:
+
+```sh
+wave run app.py
+```
+
+Which will start a Wave app at <http://localhost:10101>.
+
+## Interactive examples
+
+If you prefer learning by doing, you can run `wave fetch` command that will download all the existing small Python examples that show Wave in action. The best part is that all these examples are interactive, meaning you can edit their code directly within the browser and observe the changes.
```

Comment:
The examples are not interactive on their own. Would be good to mention the Tour app and how to start it first, then talk about the interactivity.